storage/remote/queue_manager.go

```diff
@@ -170,20 +208,16 @@ func newQueueManagerMetrics(r prometheus.Registerer, rn, e string) *queueManager
 		Help:        "The number of shards that the queues shard calculation wants to run based on the rate of samples in vs. samples out.",
 		ConstLabels: constLabels,
 	})
-	m.bytesSent = prometheus.NewCounter(prometheus.CounterOpts{
+	m.commonBytesVector = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace:   namespace,
 		Subsystem:   subsystem,
 		Name:        "sent_bytes_total",
-		Help:        "The total number of bytes sent by the queue.",
-		ConstLabels: constLabels,
-	})
-	m.maxSamplesPerSend = prometheus.NewGauge(prometheus.GaugeOpts{
-		Namespace:   namespace,
-		Subsystem:   subsystem,
-		Name:        "max_samples_per_send",
-		Help:        "The maximum number of samples to be sent, in a single request, to the remote storage.",
+		Help:        "The total number of bytes sent by the queue after compression.",
 		ConstLabels: constLabels,
-	})
+	}, []string{"type"})
+
+	m.bytesSent = m.commonBytesVector.WithLabelValues(samples)
```

Comment:
Is commonBytesVector necessary, or should the bytesSent and metadataBytesSent metrics just be created without the underlying vector?

Reply:
reworked the metrics, let me know if there's anything left there you think should change